kalite/updates/static/js/updates/bundle_modules/update_videos.js

```diff
@@ -415,10 +415,12 @@ function getSelectedStartedMetadata(data_type) {
 }
 
 function setNodeClass(node, className) {
-    if (node.extraClasses != className) {
-        node.extraClasses = className;
-        if (node.parent !== null) {
-            updateNodeClass(node.parent);
+    if (node !== null) {
```

Comment:
Two reasons:

1) the youtube_id for the video being downloaded will not represent a node in the tree, as the tree is lazy loaded, and so not all videos are present.
2) the youtube_id being passed is undefined, and so again the node is null.

This check could happen before we get to this function call, but this seemed the neatest place to check.